src/DIRAC/Resources/Storage/StorageBase.py

```diff
@@ -338,7 +338,7 @@ def constructURLFromLFN(self, lfn, withWSUrl=False):
         voLFN = lfnSplitList[1]
         # TODO comparison to Sandbox below is for backward compatibility, should
         # be removed in the next release
-        if voLFN != self.se.vo and voLFN != "SandBox" and voLFN != "Sandbox":
+        if voLFN != self.se.vo and voLFN != "SandBox" and voLFN != "Sandbox" and voLFN != "S3":
```

Comment:
I guess we cam remove `Sandbox` as we've had at least a couple of releases since 2015: https://github.com/DIRACGrid/DIRAC/commit/97d90214fe3d3053e8e4ea00e3f6e5df1d62f5a9